slurmpy/slurmpy.py

```diff
@@ -61,6 +61,9 @@
 
 __script__"""
 
+VALID_DEPENDENCY_TYPES = ["after", "afterany", "afterburstbuffer",
```

Comment:
make this a literal set. (change [ to { )

Reply:
thanks changed that.